content/events/2023-infobip-shift.md

```diff
@@ -0,0 +1,13 @@
+title = "InfoBip Shift 2023"
+date = "2023-08-39T17:00:00Z"
```

Comment:
Not sure what the date is supposed to be but this is an invalid date (note the `39`)